<launch>
    <arg name="joystick_topic" default="joy"/>
    <arg name="cmd_vel_topic" default="cmd_vel"/>
    
    <param name="joystick_topic" value="$(arg joystick_topic)" type="str" />
    <param name="cmd_vel_topic" value="$(arg cmd_vel_topic)" type="str" />
    <include file="$(find teleop_twist_joy)/launch/teleop.launch">
      <arg name="joy_config" value="xbox"/>
      <arg name="config_filepath" value="$(find fritzrobot_control)/src/joy/config/teleop_twist_joy_node.yaml"/>
    </include>
    <node pkg="fritzrobot_control" name="joy_ext" type="joy_ext"/>
    

</launch>
